packages/components/src/style.scss

```diff
@@ -1,3 +1,6 @@
+$components-color-accent: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
+$components-color-accent-darker-10: var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #006ba1));
+$components-color-accent-darker-20: var(--wp-components-color-accent-darker-20, var(--wp-admin-theme-color-darker-20, #005a87));
```

Comment:
We could probably look into moving these variables to a separate file

Reply:
Done in https://github.com/WordPress/gutenberg/pull/44668/commits/d3963987686ba42dd7fd3d76db3cc9a377c8f6e3